### DISCUSSION TITLE
PayPal Option Missing On Checkout Page

### DISCUSSION THREAD

**Discussion by jfidelsolutions** [woocommerce/woocommerce#45046] [Checkout Flow]
Hello!

I have PayPal Checkout enabled in my WooCommerce Payment settings, but there's no PayPal button/option on the checkout page. There's only the option to enter one's credit card information. Please help!! - Thanks

**Comment by nerrad** [comment]
Hi @jfidelsolutions 

The Cart and Checkout blocks are still in preview and there are a number of extensions and payment methods that are not integrated yet. Paypal Checkout is one of those extensions. You can follow [this document](https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/) to keep up to date with the latest status of the blocks.

I'm going to close this issue as there's nothing actionable for it at the moment. We are aware of there being limited payment method support and it's on our roadmap to integrate WooCommerce owned extensions as well as encourage third party developers to integrate their own extensions.

Thanks for taking the time to create the issue and make sure we were aware!
